package/Dockerfile

```
@@ -379,11 +380,12 @@ ARG VERSION
 ARG TAGS="k8s"
 ARG LINKFLAGS="-extldflags -static -s"
 ARG LDFLAGS="-X main.VERSION=${VERSION} $LINKFLAGS"
+ARG EXTRAFLAGS=""
```

The default is already empty, no need for it.

```suggestion
ARG GCFLAGS
```